cleanlab/rank.py

```diff
@@ -556,52 +555,87 @@ def get_confidence_weighted_entropy_for_each_label(
     return label_quality_scores
 
 
-def get_knn_distance_ood_scores(
-    features: np.ndarray, nbrs: NearestNeighbors, k: int = None
+def get_outlier_scores(
+    features: Optional[np.ndarray] = None,
+    knn: Optional[NearestNeighbors] = None,
+    k: int = 10,
+    t: int = 1,
 ) -> np.ndarray:
-    """Returns the KNN distance out-of-distribution (OOD) score for each datapoint.
+    """Returns the KNN distance outlier score for each example.
 
-    This is a function to compute OOD scores where higher scores indicate the datapoint is more likely to be OOD.
+    Returns an outlier score for each example based on its feature values, where higher scores indicate examples more
+    likely to be outliers. The score is based on the average distance between the example and its
+    K nearest neighbors in the dataset (in feature space).
```

Comment:
Add sentence:

Scores lie in [0,1] with values near 0 indicating examples that are less typical in the dataset distribution.